content/topics/solo-learn/sql-intermediate/3-working-with-data/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+content_type: project
+flavours:
+- none
+prerequisites:
+  hard:
+  - topics/solo-learn/sql-intermediate/2-table-constraints
+  soft: []
+ready: true
+submission_type: link
+tags:
+- sql
+- solo-learn
+title: Solo Learn - SQL Intermediate - 4 Working with Data

```

Comment:
The file name says '3-working-with-data' but the title says '4 Working with data'.

Please double check file names